dallinger/prolific.py

```diff
@@ -566,7 +572,7 @@ def _req(self, method: str, endpoint: str, **kw) -> dict:
                     response = {
                         "results": [
                             {
-                                "id": "62fce6fff0a78eb4f3ebc09c",
+                                "id": self.workspace_id,
```

Comment:
Can't you just update all these cases of `self.workspace_id` with `config.get("prolific_workspace")`? Then you don't need to override `validate_workspace`?

Reply:
Only if I have a valid ID set as `prolific_workspace`. I.e. the name of a  workspace does not work!

Reply:
OK, you could update the "title" field with `prolific_workspace` then?

Reply:
I get something like this:
```
❯❯ Error accessing http://localhost:5000/launch (500):
{"status": "error", "message": "Failed to open recruitment, check experiment server log for details: Error finding or creating specified project: Simulated Prolific API call could not be matched:\nmethod: GET\nendpoint: /workspaces/Workspace Pilot/projects/\nkw: {}"}
```

Reply:
Presumably that means the workspace ID translation code is not working?

Reply:
I think, it is because the workspace name is used in the fake API responses which are used for subsequent calls, where the URL then doesn't match the regex, because it expects a hexadecimal value.

Reply:
Ok, well maybe the best thing is to inspect the config variable, check the format, and then assign it to id or title accordingly?